pkg/lib/event/store.go

```diff
@@ -7,12 +7,12 @@ import (
 )
 
 type StoreImpl struct {
-	SQLBuilder  *appdb.SQLBuilder
+	SQLBuilder  *appdb.SQLBuilderApp
 	SQLExecutor *appdb.SQLExecutor
 }
 
 func (s *StoreImpl) NextSequenceNumber() (seq int64, err error) {
-	builder := s.SQLBuilder.Global().
+	builder := s.SQLBuilder.
```

Comment:
This seems incorrect. Select the next value from the sequence should not be app-aware.

Reply:
Updated!